orcid-persistence/src/main/java/org/orcid/persistence/dao/impl/ProfileDaoImpl.java

```diff
@@ -55,22 +55,51 @@ public void remove(String id) {
         super.remove(id);
     }
 
+    /**
+     * Get a list of the ORCID id's with the given indexing status
+     * @param indexingStatuses
+     *          The list of desired indexing status
+     * @param maxResults
+     *          Max number of results
+     * @return a list of object arrays where the object[0] contains the orcid id and object[1] contains the indexing status                           
+     * */
     @SuppressWarnings("unchecked")
-    public List<String> findOrcidsByIndexingStatus(IndexingStatus indexingStatus, int maxResults) {
+    @Override
+    public List<Object[]> findOrcidsByIndexingStatus(IndexingStatus indexingStatus, int maxResults) {
```

Comment:
@TomDemeranville we will return the orcid id and the indexing status for the record, so, we can check on processProfilePendingIndexingInTransaction what kind of request it is, so, we send to MQ only REINDEX requests